unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/SceneController.cs

```diff
@@ -352,8 +352,11 @@ public bool ProcessMessage(QueuedSceneMessage_Scene msgObject, out CustomYieldIn
             }
             catch (Exception e)
             {
-                throw new Exception(
-                    $"Scene message error. scene: {scene.sceneData.id} method: {method} payload: {JsonUtility.ToJson(msgPayload)} {e}");
+                Exception exception = new Exception(
```

Comment:
Should we have exception handling within the MessageBus when processing individual messages rather than relying on the specific controller?

Reply:
Yeah, exception handling on a higher level is a good idea.
I didn't want to fiddle too much with this system so I took this simple approach.